R/get_hpo.R

```diff
@@ -0,0 +1,43 @@
+#' @describeIn get_ get_
```

Comment:
I realized that adding this function here is actually not ideal, because it conflicts with the namespace in HPOExplorer.
```
1: replacing previous import ‘HPOExplorer::get_hpo’ by ‘KGExplorer::get_hpo’ when loading ‘MSTExplorer’ 
```

Instead, just use `get_ontology("hpo")` when trying to get the HPO within KGExplorer.

I'll apply this fix in the next version.